docs/submodule.md

```
@@ -0,0 +1,90 @@
+# Git Submodule 가이드
+
+> 설정 파일(application.yml)을 private 레포에서 관리하기 위한 서브모듈 사용 가이드
+
+## 구조
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)